_data/changelogs/tokens.yml

```diff
@@ -0,0 +1,14 @@
+title: Tokens
+type: token
+changelogURL:
+items:
+  - date: 2023-11-28
+    summary: Updated token references in "Using design tokens" section.
+    affectsGuidance: true
+    githubPr: 2366
+    githubRepo: uswds-site
+  - date: 2023-11-28
+    summary: Updated the settings configuration code example.
```

Comment:
## Optional - change logs

If we wanted to consolidate these into one change log, we could do something along the lines of

```suggestion
  - date: 2023-11-28
    summary: Updated token references and settings configuration example to match the current USWDS implementation. 
```